x-pack/plugins/reporting/server/types.ts

```diff
@@ -34,6 +30,8 @@ import type {
 import type { UsageCollectionSetup } from '@kbn/usage-collection-plugin/server';
 
 import { ExportTypesRegistry } from '@kbn/reporting-server/export_types_registry';
+import type { SecurityServiceStart } from '@kbn/core-security-server';
```

Comment:
This will not be used in this file, just for the `ReportingInternalStart` interface in `reporting/server/core.ts`:
```suggestion
import type { SecurityServiceStart } from '@kbn/core-security-server';
```